### DISCUSSION TITLE
Multiple I2S errors on compile.

### DISCUSSION THREAD

**Discussion by fabiendostie** [marcel-licence/esp32_fm_synth#11] [Q&A]
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/esp32_fm_synth.ino: In function 'void MidiSyncMasterLoop()':
esp32_fm_synth:297:9: error: 'Midi_SyncRecvd' was not declared in this scope
         Midi_SyncRecvd();
         ^~~~~~~~~~~~~~
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/esp32_fm_synth.ino:297:9: note: suggested alternative: 'Midi_SendRaw'
         Midi_SyncRecvd();
         ^~~~~~~~~~~~~~
         Midi_SendRaw
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino: In function 'bool i2s_write_stereo_samples_buff(float*, float*, int)':
i2s_interface:221:62: error: 'union i2s_write_stereo_samples_buff(float*, float*, int)::sampleTUNT' has no member named 'sample'; did you mean 'sampleTUNT'?
     i2s_write(i2s_port_number, (const char *)&sampleDataU[0].sample, 4 * buffLen, &bytes_written, portMAX_DELAY);
                                                              ^~~~~~
                                                              sampleTUNT
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino: At global scope:
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino:318:49: warning: 'I2S_COMM_FORMAT_I2S' is deprecated [-Wdeprecated-declarations]
     .communication_format = (i2s_comm_format_t)(I2S_COMM_FORMAT_I2S | I2S_COMM_FORMAT_I2S_MSB),
                                                 ^~~~~~~~~~~~~~~~~~~
In file included from /Users/lefab/Documents/Arduino/hardware/espressif/esp32/tools/sdk/esp32/include/driver/include/driver/i2s.h:16,
                 from /Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino:45:
/Users/lefab/Documents/Arduino/hardware/espressif/esp32/tools/sdk/esp32/include/hal/include/hal/i2s_types.h:95:5: note: declared here
     I2S_COMM_FORMAT_I2S       __attribute__((deprecated)) = 0x01, /*!< I2S communication format I2S, correspond to `I2S_COMM_FORMAT_STAND_I2S`*/
     ^~~~~~~~~~~~~~~~~~~
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino:318:71: warning: 'I2S_COMM_FORMAT_I2S_MSB' is deprecated [-Wdeprecated-declarations]
     .communication_format = (i2s_comm_format_t)(I2S_COMM_FORMAT_I2S | I2S_COMM_FORMAT_I2S_MSB),
                                                                       ^~~~~~~~~~~~~~~~~~~~~~~
In file included from /Users/lefab/Documents/Arduino/hardware/espressif/esp32/tools/sdk/esp32/include/driver/include/driver/i2s.h:16,
                 from /Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino:45:
/Users/lefab/Documents/Arduino/hardware/espressif/esp32/tools/sdk/esp32/include/hal/include/hal/i2s_types.h:96:5: note: declared here
     I2S_COMM_FORMAT_I2S_MSB   __attribute__((deprecated)) = 0x01, /*!< I2S format MSB, (I2S_COMM_FORMAT_I2S |I2S_COMM_FORMAT_I2S_MSB) correspond to `I2S_COMM_FORMAT_STAND_I2S`*/
     ^~~~~~~~~~~~~~~~~~~~~~~
i2s_interface:342:19: error: 'I2S_BCLK_PIN' was not declared in this scope
     .bck_io_num = I2S_BCLK_PIN,
                   ^~~~~~~~~~~~
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino:342:19: note: suggested alternative: 'I2S_CLK_EN'
     .bck_io_num = I2S_BCLK_PIN,
                   ^~~~~~~~~~~~
                   I2S_CLK_EN
i2s_interface:343:19: error: 'I2S_WCLK_PIN' was not declared in this scope
     .ws_io_num =  I2S_WCLK_PIN,
                   ^~~~~~~~~~~~
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino:343:19: note: suggested alternative: 'I2S_CLK_EN'
     .ws_io_num =  I2S_WCLK_PIN,
                   ^~~~~~~~~~~~
                   I2S_CLK_EN
i2s_interface:344:21: error: 'I2S_DOUT_PIN' was not declared in this scope
     .data_out_num = I2S_DOUT_PIN,
                     ^~~~~~~~~~~~
/Users/lefab/Documents/Arduino/esp32/esp32_fm_synth/i2s_interface.ino:344:21: note: suggested alternative: 'I2S_OUT_RST'
     .data_out_num = I2S_DOUT_PIN,
                     ^~~~~~~~~~~~
                     I2S_OUT_RST
Multiple libraries were found for "WiFi.h"
 Used: /Users/lefab/Documents/Arduino/hardware/espressif/esp32/libraries/WiFi
 Not used: /Applications/Arduino.app/Contents/Java/libraries/WiFi
exit status 1
'Midi_SyncRecvd' was not declared in this scope

**Comment by marcel-licence** [comment]
Is it okay to delete this on continue in https://github.com/marcel-licence/esp32_fm_synth/discussions/12 ?
